mod/fabric/src/main/resources/mapsync.fabric.mixins.json

```diff
@@ -0,0 +1,14 @@
+{
+	"required": true,
+	"minVersion": "0.8",
+	"package": "gjum.minecraft.mapsync.fabric.mixins",
+	"compatibilityLevel": "JAVA_17",
+	"mixins": [],
+	"client": [
+
+	],
+	"server": [],
```

Comment:
This is empty, why have separate {fabric,forge} mixin files?

Reply:
It's just for future use cases where we may have mixins for fabric-only mods or forge-only mods. For example, ModMenu is a fabric-only mod and so the dependency is only in the fabric subproject. If hooking into ModMenu required a mixin, that mixin would be better placed in the fabric subproject, rather than common.